server/services/return-of-service.js

```diff
@@ -122,17 +142,26 @@ const updateReturnOfServiceStatus = async ({ participantId, newSite, newDate, ne
     throw new Error(rosError.noStartDate);
   }
 
+  const updatedData = {
+    site_id: newSiteId || rosParticipant.site_id,
+    data: {
+      ...rosParticipant.data,
+      date: newDate || rosParticipant.data.date,
+      startDate: newStartDate || rosParticipant.data.startDate,
+    },
+  };
   dbClient.db[collections.ROS_STATUS].update(
     { participant_id: participantId, is_current: true },
-    {
-      site_id: newSite || rosParticipant.site_id,
-      data: {
-        ...rosParticipant.data,
-        date: newDate || rosParticipant.data.date,
-        startDate: newStartDate || rosParticipant.data.startDate,
-      },
-    }
+    updatedData
   );
+  dbClient.db[collections.MOH_EDIT_AUDIT].insert({
+    user: user || 'system',
+    data: {
+      participantId,
+      oldData: { ...rosParticipant },
```

Comment:
In other parts of the codebase where we track changes we use `from` and `to`, would it be more consistent to apply that naming here?

https://github.com/bcgov/hcap/blob/17fd7f9dcd9085c2d28c375bae1ca78600de55ab/server/services/participants.js#L136